units/tests/experimental_test/operations.yml

```diff
@@ -85,3 +85,4 @@ disable-tls-tcp-routing-windows-stage-2-route-emitter.yml:
   - ../windows2019-cell.yml
   - enable-nginx-routing-integrity-windows2019.yml
   - disable-tls-tcp-routing-windows-stage-1-unproxied-ports.yml
+use-noble-stemcell: {}
```

Comment:
I think this should actually have the `.yml` ending, e.g. `use-noble-stemcell.yml`

Reply:
Done.